Podfile.lock

```diff
@@ -182,7 +182,7 @@ DEPENDENCIES:
   - WordPressCom-Analytics-iOS (= 0.1.22)
   - WordPressCom-Stats-iOS (= 0.8.1)
   - WordPressComKit (from `https://github.com/Automattic/WordPressComKit.git`, tag `0.0.6`)
-  - WPMediaPicker (~> 0.10.3)
+  - WPMediaPicker (from `https://github.com/wordpress-mobile/MediaPicker-iOS.git`, commit `04239494418da7d1937b4f59dcf68cc805dece5b`)
```

Comment:
This is for testing of the new release before I create the new pod version. If all is ok I will release the pod version and change this to the version number.